src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx

```diff
@@ -10,16 +10,28 @@ metaDescription: "Notifications are where you integrate your New Relic data with
 ---
 
 Notifications are a consolidation of the different ways to send notification-events to third-party services, such as Slack, Jira, ServiceNow, and email. You can also use webhooks to send your data to any compatible third-party service.
-
+ 
 Integrate with your systems by configuring destinations and message templates.
+ 
+From the API perspective, this is the diagram flow: Workflows => Channel id(s) => Destination
 
-## Destinations [#destinations]
+## Why are there 3 different configurations? [#configurations]
+
+We have separated configurations to configure a customized notification so we can provide connection details/ to authenticate once(Destination), but create many customized templates for your outgoing events, and connect these with the destination.
+ 
+The Notifications API and configuration objects are product agnostic. Workflows is one of 3 products that use the Notifications API to send notifications events / integrate with third parties, along with [Error Inbox](https://docs.newrelic.com/docs/errors-inbox/error-external-services/) and others. Having a unified platform for notifications, you can, for example, set a single Slack or Jira destinations, and use them both in a Workflow and Error-Inbox configuration
```

Comment:
I think it's called "Errors Inbox" but I could be mistaken.

The first half of this feels like it's dropping internal terms and details on the customer that they don't need to know or that will add needless complexity. Can we just communicate that these destinations can be used elsewhere, without referring to the Notifications API (which feels like a bit of sausage-making)?

Reply:
also, it's 'errors inbox', not capitalized...